lib/utils/Thing.js

```diff
@@ -129,7 +134,7 @@ export default class Thing {
 	}
 
 	isComment() {
-		return this.thing.classList.contains('comment');
+		return this.thing.classList.contains('comment') || this.thing.classList.contains('was-comment');
```

Comment:
and potentially `|comment-message` too, since PMs and comment replies have different markup in the inbox.